src/makeCalculator.js

```diff
@@ -4,7 +4,53 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(num) {
+      result += num;
+
+      return this;
+    },
+
+    subtract(num) {
+      result -= num;
+
+      return this;
+    },
+
+    multiply(num) {
+      result *= num;
+
+      return this;
+    },
+
+    divide(num) {
+      if (num !== 0) {
+        result /= num;
+      } else {
+        throw new Error('Division by zero is not allowed');
+      }
```

Comment:
Good job checking for division by zero and throwing an error. This prevents runtime errors and makes the calculator more robust.